### ISSUE TITLE
More pleasing experience when viewing an article

### ISSUE THREAD

**Issue by justinGilmer** [livecomsjournal/journal_information#30]
Just some things and possible improvements I noticed between viewing an article on PLOS and on our site.

### Pros
1. The PDF viewer / integration with Firefox looks great.
    - I really like the way that is set up on our site.
    - Very pleasing to view compared to the html interactive view that defaults for PLOS
2. The longer we can prevent advertisements the better, looks very jarring when viewing PLOS.

### Cons
1. The top of the page when viewing an article on LiveCOMS seems out of place. I feel like our logo should be bolder/bordered so the user can recognize that it is a link to the landing page.
2. When scrolling through an article on plos, the title and authors travels with the scroll.
    - See [this PLOS example](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182522)
    - Compared to our [current view](http://www.livecomsjournal.org/article/2031-why-we-need-the-living-journal-of-computational-molecular-science)

### Improvements
1. Change the logo when viewing an article to look more like a link/not blending into the background
2. Have the title, authors, link back to the landing page, and a download pdf button follow the user at as they scroll through an article.
    - Add an option to remove the dynamic bar if the user would like (small x in the top right)

I am unsure how difficult/possible this is to implement with our journal.

**Comment by davidlmobley** [comment]
@mrshirts - you're the only one with edit access to the livecomsjournal.org website; can you comment on how doable these would be if at all? Or is this just feedback we should get to Scholastica?

**Comment by jchodera** [comment]
> Change the logo when viewing an article to look more like a link/not blending into the background

Alternatively, an actual link that says [Back to articles] or something would also make it very obvious what to do to go back.

**Comment by davidlmobley** [comment]
@mrshirts - this is sitting idle. Is there anything we are able to do on this or is it something we have to bring up with Scholastica?

**Comment by mrshirts** [comment]
Sorry, this did get lost!

Things we can do trivially: Put a small logo (fixed size) to the left of
"LiveCoMS" (if we had a logo), or make it say something different (like the
full name).  We can change the color as well.

Can't currently do:

* change so it's a banner image in the background.

* Have the authors names and title follow down.

Thoughts?

**Comment by dmzuckerman** [comment]
Logo is always good if we get one - and can we have this go on the little browser tab??

I like authors names and titles following - also a download link if possible.

**Comment by dmzuckerman** [comment]
Ideally, there should also be a 'How to cite' button/link.

BTW, if you look at our [self-justifying editorial](http://www.livecomsjournal.org/article/2031-why-we-need-the-living-journal-of-computational-molecular-science) on [Google Scholar](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C38&q=WhyweneedtheLivingJournalofComputationalMolecularScience&btnG=#d=gs_cit&p=&u=%2Fscholar%3Fq%3Dinfo%3Ayae5gIMW1vUJ%3Ascholar.google.com%2F%26output%3Dcite%26scirp%3D0%26hl%3Den), it doesn't seem to have a full citation in the 'cite' feature - i.e., no volume or whatever.  I guess that will change for articles going forward?

**Comment by davidlmobley** [comment]
Agree logo would be good. 

> BTW, if you look at our self-justifying editorial on Google Scholar, it doesn't seem to have a full citation in the 'cite' feature - i.e., no volume or whatever. I guess that will change for articles going forward?

Yes, right; that's (at least) because it has no volume or page numbers (though maybe there are other reasons too?). :) Maybe we should put that into the first issue so it can have them.

**Comment by mrshirts** [comment]
We haven't assigned it an issue (or created any issues), so that's why it shows up w/o issue number.  I'm investigating if we can publish an issue, and THEN add citations.

**Comment by mrshirts** [comment]
Aha, it looks like we CAN create an issue and then add afterwards.  So we can create an issue, add this in, and then have it report that.  We don't have access to Crossref yet, so we can't assign a journal specific DOI - that should come in a month or two.

**Comment by mrshirts** [comment]
>I like authors names and titles following - also a download link if possible.
Don't count on this; they would have to change it at Scholastica.

> and can we have this go on the little browser tab??

I'm not sure exactly which location you mean.

>  also a download link if possible.

A download link on the tab with the names (that doesn't exist yet)?  Just want to be clear what is wanted.

**Comment by mrshirts** [comment]
So for an issue, we need: 

* Volume (required)
* Issue Number (required)
* Year (required)

All of those are obvious!

Issue Image (required)
If we just want general fancy molecular picture for each issue, that's fine.  We can perhaps have some sort of competition longer term. 

Lede (required - 30 words)
Description of the issue (recommended)
​Title (recommended)

What would you all suggest here?  
Here's some examples of how people are using these, and where they appear on the page, though it's not clear exactly which of these text fields are  going where. 
https://www.newlibs.org/issues

Though it's not clear most journals are using thesef eatures.

**Comment by dmzuckerman** [comment]
For issue image, why not tile some of the article images?  That would be descriptive and maybe even cool.

**Comment by dmzuckerman** [comment]
> little browswer tab

the tiny icon appearing to the left of the title on a tab of your browser - as in having many tabs open

> moving download link

ideally, enabling download of article from any page in the article, without having to scroll to top ... but this should hardly be a priority!

**Comment by slochower** [comment]
> Ideally, there should also be a 'How to cite' button/link.

I was just looking for this so I can cite your article! So... +1

**Comment by davidlmobley** [comment]
> For issue image, why not tile some of the article images? That would be descriptive and maybe even cool.

I like this idea -- not necessarily as a policy (e.g. if someone has a particularly stunning graphic that we want to put on the cover that could be good too) but as something we probably do often. :)

**Comment by mrshirts** [comment]
I think the tile will be too visually busy.  Especially if included in social media posts, etc.  My $0.02.

**Comment by dmzuckerman** [comment]
I don't have strong feelings about this so if @mrshirts wants custom images, it's fine with me.

**Comment by dmzuckerman** [comment]
Does anyone know if @slochower can cite our article (editorial, I guess) now, or does he have to wait for the first issue and DOI?

**Comment by davidlmobley** [comment]
@dmzuckerman - I think for now he could cite it as if it were a website.

An easy solution, however, is to connect your GitHub repository to Zenodo and then do a "release" of your "software" (GitHub repo) which will get it a permanent DOI. Then he can cite the DOI. 

I take @mrshirts point about the tile being busy; OTOH it may be a good fallback option if no one has anything particularly suitable/wants to make real "cover art".

**Comment by mrshirts** [comment]
I will work on creating an issue - as noted above in this chain, I have to write a bunch of text, that will be featured on the site, so I actually have to think about it.

**Comment by dmzuckerman** [comment]
@slochower - which article did you want to cite ... our editorial about the journal or one of the as-yet-unpublished best practices manuscripts?

**Comment by slochower** [comment]
@dmzuckerman Your editorial and "Paper Writing as Code Development." They are aligned philosophically with a project I've been working on for collaborative scholarly writing via GitHub and continuous integration (see https://github.com/greenelab/meta-review/issues/47#issuecomment-399581126 and an in-progress manuscript on its use https://greenelab.github.io/meta-review/ )

**Comment by mrshirts** [comment]
OK, I've created an issue.  Meaning, a journal issue, not a GitHub Issue.

http://www.livecomsjournal.org/issue/654

See how this works. Google Scholar should eventually update the citation with issue and date, but I'm not sure how long it will take.

**Comment by mrshirts** [comment]
We could move the "paper writing as code development" into a blog post, rather than on GitHub.

**Comment by davidlmobley** [comment]
Ah, the paper-writing as code development one... Hmm. So does that solve @slochower 's problem, @mrshirts ? Can he then cite it? Specifically, title and authors, Volume 1, Issue 1, 2018? (And page number to be assigned later?)

**Comment by mrshirts** [comment]
Good point - there's no page number or article ID within the issue.  I'm having a hard time figuring out how to do that - I'll ask.

> Ah, the paper-writing as code development one... Hmm. 

Well, I don't think it's really substantial enough for it's own article.  We could feature it as a blog post, and it could be cited as web site. . .

**Comment by davidlmobley** [comment]
Ah, right. I was thinking of the "Why LiveCoMS" article.

Yes, probably should just be a blog post.
